ProjectSidewalk/SidewalkWebpage
# Resolves #578 (partially): lazy-play landing page videos

Resolves #578 
(Partially)

- Landing page videos now pause when out of view
- New instruction video cycling code (those videos no longer cycle when out of view)

To test;
- Ensure that landing page videos still appear to be always playing (users shouldn't see them paused)
- Ensure that instruction videos (1/2/3) still cycle
- Observe at least some performance gain
